enterprise/frontend/src/embedding-sdk/components/public/CreateQuestion/CreateQuestion.tsx

```diff
@@ -1,23 +1,115 @@
-import { QuestionEditor } from "embedding-sdk/components/private/QuestionEditor";
+import { useState } from "react";
 
-import type { InteractiveQuestionProps } from "../InteractiveQuestion";
+import { FlexibleSizeComponent } from "embedding-sdk";
+import { useInteractiveQuestionContext } from "embedding-sdk/components/private/InteractiveQuestion/context";
+import { SaveQuestionModal } from "metabase/containers/SaveQuestionModal";
+import { Button, Flex } from "metabase/ui";
 
-type CreateQuestionProps = Omit<InteractiveQuestionProps, "questionId">;
+import {
+  InteractiveQuestion,
+  type InteractiveQuestionProps,
+} from "../InteractiveQuestion";
+
+type CreateQuestionProps = Partial<
+  Omit<InteractiveQuestionProps, "questionId" | "children">
+>;
```

Comment:
Same prop as `InteractiveQuestion`, but without `questionId` and `children`. All props are optional.